cypress/e2e/po/pages/home.po.ts

```diff
@@ -18,10 +20,94 @@ export default class HomePagePo extends PagePo {
   }
 
   title(): Cypress.Chainable<string> {
-    return this.self().getId('banner-title').invoke('text');
+    return cy.getId('banner-title').invoke('text');
   }
 
-  changelog(): Cypress.Chainable<string> {
-    return this.self().getId('changelog-banner').invoke('text');
+  graphicBanner(): Cypress.Chainable {
+    return cy.getId('home-banner-graphic');
```

Comment:
you're gonna hate all these comments.... but they'll really help later on when the ui changes

can you make this a banner-graphic.po?

Reply:
which can also contain graphicBannerCloseButton below